packages/cli-plugin-scaffold-app-graphql/index.js

```diff
@@ -0,0 +1,249 @@
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+const ncp = util.promisify(require("ncp").ncp);
+const findUp = require("find-up");
+const kebabCase = require("lodash.kebabcase");
+const pluralize = require("pluralize");
+const Case = require("case");
+const {replaceInPath} = require("replace-in-path");
+const fastGlob = require('fast-glob');
+
+const appsPluginsLocation = "apps/admin/src/plugins";
+
+const createPackageLocation = name => {
+	return `${appsPluginsLocation}/${name}`;
+};
+
+const readApiPackageModelName = (file) => {
+	const fileContent = fs
+		.readFileSync(file, {
+			encoding: "utf-8",
+			flag: "r"
+		})
+		.toString();
+	if(!fileContent) {
+		throw new Error(`Could not load any content from file ${file}`);
+	}
+	
+	const nameMatched = fileContent.match(/withName\("([a-zA-Z]+)"\)/);
+	if(!nameMatched) {
+		throw new Error(
+			`Could not find withName() in ${file} which is needed to detect model name.`
+		);
+	}
+	return nameMatched[1];
+};
+
+const findDataModels = (location) => {
+	const target = path.resolve(`${location}/**/*.model.ts`);
+	const files = fastGlob
+		.sync(target, {
+			unique: true,
+		});
+	if(files.length === 0) {
+		throw new Error(`Could not find any models with fast-glob pattern "${target}"`);
+	}
+	return files.map(file => ({
+		fileName: path.basename(file),
+		filePath: file,
+		modelName: readApiPackageModelName(file),
+	}))
+};
+
+module.exports = [
+	{
+		name: "cli-plugin-scaffold-template-graphql-app",
+		type: "cli-plugin-scaffold-template",
+		scaffold: {
+			name: "GraphQL Apollo Service App",
+			questions: () => {
+				return [
+					{
+						name: "apiLocation",
+						message: "Enter existing GraphQL API location (relative to the project root) or empty for none",
+						validate: apiLocation => {
+							if(!apiLocation) {
+								return true;
+							}
+							if(fs.existsSync(apiLocation) === false) {
+								return "There is no GraphQL API in given location";
+							}
+							try {
+								findDataModels(apiLocation);
+							}
+							catch (ex) {
+								return `Could not find existing API model in ${apiLocation}, error: ${ex.message}`;
```

Comment:
No need to append the `ex.message`, because it will create a very long and not very clear error message. User shouldn't care about `glob patterns`, etc. So let's keep it short: `Couldn't find existing data model in ${apiLocation}` - short and clear 👍.

I mentioned this further in the review, but pick one way of saying something. Is it `API model`? `data model`? `entity`? Pick one and use it everywhere.